### PR TITLE
Active Patterns and nuget package reference

### PR DESCRIPTION
**Pull request by SpiralOSS**
- Added single-case active pattern examples
- Added additional multi-case active pattern example
- Added referencing a nuget package

### CODE REVIEW

File: docs/fsharp-cheatsheet.md [hunk 1]
```diff
@@ -404,6 +404,31 @@ Another way of implementing interfaces is to use *object expressions*.
...
+*Single-case active patterns*:
+
+    // Basic
+    let (|EmailDomain|) (email : string) =
+        let parts = email.Split '@'
+        parts[1]
+    let (EmailDomain emailDomain) = "yennefer@aretuza.org"  // emailDomain = 'aretuza.org'
+
+    // As Parameters
+    let (|Real|) (x:System.Numerics.Complex) =
```
Comment (bartelink) [inline docs/fsharp-cheatsheet.md:416]:
```suggestion
    let (|Real|) (x: System.Numerics.Complex) =
```

File: docs/fsharp-cheatsheet.md [hunk 2]
```diff
@@ -404,6 +404,31 @@ Another way of implementing interfaces is to use *object expressions*.
 
 <a name="ActivePatterns"></a>Active Patterns
 ---------------
+*Single-case active patterns*:
+
+    // Basic
+    let (|EmailDomain|) (email : string) =
```
Comment (bartelink) [inline docs/fsharp-cheatsheet.md:410]:
```suggestion
    let (|EmailDomain|) (email: string) =
```

**Comment by bartelink** [comment]
@dungpa .. are you still maintaining this?

**Comment by dungpa** [comment]
@bartelink Not really.
Is there any good place where I can transfer ownership to someone interested in maintaining this?

**Comment by bartelink** [comment]
I guess fsprojects might make sense?

Like the rest of the world, I'm not personally actively looking for things to take care of given I already maintain and/or lurk on too many projects...

While I appreciated this cheat sheet a lot when I was fresh here, I'm not sure I'd be the best maintainer as:
- arguably the improvements in the MS docs have made cheat sheets less relevant (though there's things a cheat sheet can achieve that official docs by definition can't do)
- I am not in the target audience (not learning; not teaching all that much)

@SpiralOSS I guess this is the last thing you expected, but would you consider it if it got moved there?

If we ping sergey-tihon, I believe he can rig things, assuming we meet the critiria (I could be a largely silent helper)

Alternately, maybe, given current traffic levels on this, the best thing might be to consider such moves only next time it arises?

**Comment by SpiralOSS** [comment]
Sure, I can take it on. Honestly, I'll have a learning curve, but I guess this is the best way to learn.

**Comment by dsyme** [comment]
@bartelink @SpiralOSS My suggestion is to fork/clone this per fsprojects policy, see https://github.com/fsprojects/.github/issues/47#issue-1939922057

**Comment by dsyme** [comment]
Now transferred!

@dungpa you are admin. Please add someone as 2nd maintenaner? I suggest @bartelink?

**Pull request merged**